app/javascript/dashboard/assets/scss/_variables.scss

```diff
@@ -99,3 +99,7 @@ $ionicons-font-path: '~ionicons/fonts';
 
 // Transitions
 $transition-ease-in: all 0.250s ease-in;
+
+:root {
+  --dashboard-app-tabs-height: 3.9rem;
```

Comment:
38px + 1px for the bottom border.

Reply:
![Screenshot2022-07-08 at 11 51 12](https://user-images.githubusercontent.com/15716057/177929733-f8a793c1-1558-4bf6-a923-473f0128bd73.gif)

I think it should be 38px, 39 is adding a small 1px space between the border and tab indicator.